packages/sdk-ts/src/client/chain/transformers/ChainGrpcCommonTransformer.ts

```
@@ -0,0 +1,11 @@
+import { Coin } from '@injectivelabs/ts-types'
+import { GrpcCoin } from '../../../types/index.js'
+
+export class ChainGrpcCommonTransformer {
+  static grpcCoinToCoin(coin: GrpcCoin): Coin {
+    return {
+      denom: coin.denom,
+      amount: coin.amount,
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)